ai/prompt_context/transform_txt.py

```diff
@@ -0,0 +1,117 @@
+"""
+Turns the extracted text into a format ideal for LLM consumption.
+
+Pre-requisite:
+
+Run `python transform_html.py` first to extract the text from the HTML files.
+"""
+
+import os
+import time
+
+import google.generativeai as genai
+
+# Note: this prompt does not work well for every file
+# in particular, it omits too much information from the style API.
+# I use this to cleanup the style doc:
+# "Clean up the formatting for the following Python docs to make it easier for an LLM to understand:"
```

Comment:
So need to manually run code to transform the style doc?

Reply:
yup that's right. I just plugged it into AI studio and copy-pasted, which isn't ideal, but I think we can rework this prompt context pipeline.